collections/_posts/2024-12-02-SSL_comparison.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: review
+title: "Systematic comparison of semi-supervised and self-supervised learning for medical image classification"
+tags: classification, semi-supervision, self-supervision
+author: "Juliette Moreau"
+cite:
+    authors: "Zhe Huang, Ruije Jiang, Shuchin Aeron and Michael C. Hughes"
+    title:   "Systematic comparison of semi-supervised and self-supervised learning for medical image classification"
+    venue:   "CVPR, 2024"
+pdf: "https://arxiv.org/pdf/2307.08919"
+---
+
+
+# Highlights
+
+* Benchmark of SSL methods on medical images.
+* Implementation of a real life hyperparameters optimization.
+* Comparison of semi-supervised and self-supervised learning.
+
+# Note
+
+Code is available here: https://github.com/tufts-ml/SSL-vs-SSL-benchmark
```

Comment:
It would convenient for the link to be clickable :wink: 
```suggestion
Code is available here: [https://github.com/tufts-ml/SSL-vs-SSL-benchmark](https://github.com/tufts-ml/SSL-vs-SSL-benchmark)
```